otel-infrastructure-collector/k8s-helm/Chart.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: v2
 name: otel-infrastructure-collector
 description: OpenTelemetry Infrastructure collector
-version: 0.1.2
+version: 0.1.3
 keywords:
   - OpenTelemetry Collector
   - OpenTelemetry Infrastructure Collector
   - Coralogix
 dependencies:
   - name: opentelemetry-collector
-    version: "0.55.0"
-    repository: https://open-telemetry.github.io/opentelemetry-helm-charts
+    version: "0.69.0"
+    repository: https://cgx.jfrog.io/artifactory/coralogix-charts-virtual
```

Comment:
I wonder if there's anything changing the upstream chart to look out for, but I'm not sure. Probably if the smoke tests pass we should be safe.